httpserver/http_connection.cpp

```diff
@@ -1081,6 +1095,38 @@ void connection_t::on_get_stats() {
     this->response_.result(http::status::ok);
 }
 
+void connection_t::on_get_logs() {
+
+    /// Limit this call to 1 request per second
+    static time_t last_req_time = 0;
+    const time_t now = time(nullptr);
+    constexpr time_t PERIOD = 1;
+
+    if (now - last_req_time < PERIOD) {
+        this->body_stream_ << "Too many request, try again later.";
+        this->response_.result(http::status::too_many_requests);
+        return;
+    }
+
+    last_req_time = now;
+
+    auto dev_sink = dynamic_cast<loki::dev_sink_mt*>(
+        spdlog::get("loki_logger")->sinks()[2].get());
+
+    if (dev_sink == nullptr) {
+        LOKI_LOG(critical, "Sink #3 should be dev sink");
+        assert(false);
```

Comment:
Just triple checking that this will not be in the production binary

Reply:
It won't be!